File: server/src/main/java/message/csmessage/uservalidation/CsLogin.java
package message.csmessage.uservalidation;

import com.fasterxml.jackson.annotation.JsonCreator;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import message.csmessage.CsMessage;
import session.Session;

@AllArgsConstructor
public class CsLogin extends CsMessage {
    public String userName;
    public String password;

    @JsonCreator
    public CsLogin(@JsonProperty("generatedTime") long generatedTime, @JsonProperty("userName") String userName, @JsonProperty("password") String password) {
        this.generatedTime = generatedTime;
        this.userName = userName;
        this.password = password;
    }

    @Override
    public void onMessage(Session session) {
        session.onLogin(userName, password);
    }
}

File: Client/Fruitster/Assets/Script/NetConnection/Messages/CsMessages/UserValidation/CsLogin.cs
using System;

namespace Script.Messages.CsMessages
{
    [Serializable]
    public class CsLogin : CsMessage
    {
        public string userName;
        public string password;

        public CsLogin(string userName, string password)
        {
            this.userName = userName;
            this.password = password;
            UserProperties.MainPlayer.userName = userName;
        }
    }
}